### PR TITLE
Documentation changes for version 4 of the AssembleViews connector

### PR DESCRIPTION
**Pull request by moneal37**

### CODE REVIEW

**Comment by moneal37** [comment]
> @microsoft-github-policy-service agree [company="Autodesk"]

**Comment by cmcclister** [comment]
@bezhan-msft  please advise when you have a chance to review and approve this PR for merge. Thanks! cc @JasonWHowell @moneal37

**Comment by moneal37** [comment]
@microsoft-github-policy-service agree [company="Autodesk"]

> @moneal37 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement
> ### Contribution License Agreement
> This Contribution License Agreement (**“Agreement”**) is agreed to by the party signing below (**“You”**), and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Your contributions to Microsoft open source projects. This Agreement is effective as of the latest signature date below.
> 
> 1. **Definitions**.
>    **“Code”** means the computer software code, whether in human-readable or machine-executable form,
>    that is delivered by You to Microsoft under this Agreement.
>    **“Project”** means any of the projects owned or managed by Microsoft and offered under a license
>    approved by the Open Source Initiative ([www.opensource.org](http://www.opensource.org)).
>    **“Submit”** is the act of uploading, submitting, transmitting, or distributing code or other content to any
>    Project, including but not limited to communication on electronic mailing lists, source code control
>    systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
>    discussing and improving that Project, but excluding communication that is conspicuously marked or
>    otherwise designated in writing by You as “Not a Submission.”
>    **“Submission”** means the Code and any other copyrightable material Submitted by You, including any
>    associated comments and documentation.
> 2. **Your Submission**. You must agree to the terms of this Agreement before making a Submission to any
>    Project. This Agreement covers any and all Submissions that You, now or in the future (except as
>    described in Section 4 below), Submit to any Project.
> 3. **Originality of Work**. You represent that each of Your Submissions is entirely Your original work.
>    Should You wish to Submit materials that are not Your original work, You may Submit them separately
>    to the Project if You (a) retain all copyright and license information that was in the materials as You
>    received them, (b) in the description accompanying Your Submission, include the phrase “Submission
>    containing materials of a third party:” followed by the names of the third party and any licenses or other
>    restrictions of which You are aware, and (c) follow any other instructions in the Project’s written
>    guidelines concerning Submissions.
> 4. **Your Employer**. References to “employer” in this Agreement include Your employer or anyone else
>    for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
>    Submission is made in the course of Your work for an employer or Your employer has intellectual
>    property rights in Your Submission by contract or applicable law, You must secure permission from Your
>    employer to make the Submission before signing this Agreement. In that case, the term “You” in this
>    Agreement will refer to You and the employer collectively. If You change employers in the future and
>    desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
>    and secure permission from the new employer before Submitting those Submissions.
> 5. **Licenses**.
> 
> * **Copyright License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
>   Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
>   the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
>   parties.
> * **Patent License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
>   Your patent claims that are necessarily infringed by the Submission or the combination of the
>   Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
>   import or otherwise dispose of the Submission alone or with the Project.
> * **Other Rights Reserved**. Each party reserves all rights not expressly granted in this Agreement.
>   No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
>   granted by implication, exhaustion, estoppel or otherwise.
> 
> 6. **Representations and Warranties**. You represent that You are legally entitled to grant the above
>    licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
>    have disclosed under Section 3). You represent that You have secured permission from Your employer to
>    make the Submission in cases where Your Submission is made in the course of Your work for Your
>    employer or Your employer has intellectual property rights in Your Submission by contract or applicable
>    law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
>    have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
>    You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
>    REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
>    EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
>    PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
>    NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
> 7. **Notice to Microsoft**. You agree to notify Microsoft in writing of any facts or circumstances of which
>    You later become aware that would make Your representations in this Agreement inaccurate in any
>    respect.
> 8. **Information about Submissions**. You agree that contributions to Projects and information about
>    contributions may be maintained indefinitely and disclosed publicly, including Your name and other
>    information that You submit with Your Submission.
> 9. **Governing Law/Jurisdiction**. This Agreement is governed by the laws of the State of Washington, and
>    the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
>    Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
>    exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
>    defenses of lack of personal jurisdiction and forum non-conveniens.
> 10. **Entire Agreement/Assignment**. This Agreement is the entire agreement between the parties, and
>     supersedes any and all prior agreements, understandings or communications, written or oral, between
>     the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.

**Comment by moneal37** [comment]
@microsoft-github-policy-service agree company="Autodesk"

> @moneal37 please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement
> ### Contribution License Agreement
> This Contribution License Agreement (**“Agreement”**) is agreed to by the party signing below (**“You”**), and conveys certain license rights to Microsoft Corporation and its affiliates (“Microsoft”) for Your contributions to Microsoft open source projects. This Agreement is effective as of the latest signature date below.
> 
> 1. **Definitions**.
>    **“Code”** means the computer software code, whether in human-readable or machine-executable form,
>    that is delivered by You to Microsoft under this Agreement.
>    **“Project”** means any of the projects owned or managed by Microsoft and offered under a license
>    approved by the Open Source Initiative ([www.opensource.org](http://www.opensource.org)).
>    **“Submit”** is the act of uploading, submitting, transmitting, or distributing code or other content to any
>    Project, including but not limited to communication on electronic mailing lists, source code control
>    systems, and issue tracking systems that are managed by, or on behalf of, the Project for the purpose of
>    discussing and improving that Project, but excluding communication that is conspicuously marked or
>    otherwise designated in writing by You as “Not a Submission.”
>    **“Submission”** means the Code and any other copyrightable material Submitted by You, including any
>    associated comments and documentation.
> 2. **Your Submission**. You must agree to the terms of this Agreement before making a Submission to any
>    Project. This Agreement covers any and all Submissions that You, now or in the future (except as
>    described in Section 4 below), Submit to any Project.
> 3. **Originality of Work**. You represent that each of Your Submissions is entirely Your original work.
>    Should You wish to Submit materials that are not Your original work, You may Submit them separately
>    to the Project if You (a) retain all copyright and license information that was in the materials as You
>    received them, (b) in the description accompanying Your Submission, include the phrase “Submission
>    containing materials of a third party:” followed by the names of the third party and any licenses or other
>    restrictions of which You are aware, and (c) follow any other instructions in the Project’s written
>    guidelines concerning Submissions.
> 4. **Your Employer**. References to “employer” in this Agreement include Your employer or anyone else
>    for whom You are acting in making Your Submission, e.g. as a contractor, vendor, or agent. If Your
>    Submission is made in the course of Your work for an employer or Your employer has intellectual
>    property rights in Your Submission by contract or applicable law, You must secure permission from Your
>    employer to make the Submission before signing this Agreement. In that case, the term “You” in this
>    Agreement will refer to You and the employer collectively. If You change employers in the future and
>    desire to Submit additional Submissions for the new employer, then You agree to sign a new Agreement
>    and secure permission from the new employer before Submitting those Submissions.
> 5. **Licenses**.
> 
> * **Copyright License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license in the
>   Submission to reproduce, prepare derivative works of, publicly display, publicly perform, and distribute
>   the Submission and such derivative works, and to sublicense any or all of the foregoing rights to third
>   parties.
> * **Patent License**. You grant Microsoft, and those who receive the Submission directly or
>   indirectly from Microsoft, a perpetual, worldwide, non-exclusive, royalty-free, irrevocable license under
>   Your patent claims that are necessarily infringed by the Submission or the combination of the
>   Submission with the Project to which it was Submitted to make, have made, use, offer to sell, sell and
>   import or otherwise dispose of the Submission alone or with the Project.
> * **Other Rights Reserved**. Each party reserves all rights not expressly granted in this Agreement.
>   No additional licenses or rights whatsoever (including, without limitation, any implied licenses) are
>   granted by implication, exhaustion, estoppel or otherwise.
> 
> 6. **Representations and Warranties**. You represent that You are legally entitled to grant the above
>    licenses. You represent that each of Your Submissions is entirely Your original work (except as You may
>    have disclosed under Section 3). You represent that You have secured permission from Your employer to
>    make the Submission in cases where Your Submission is made in the course of Your work for Your
>    employer or Your employer has intellectual property rights in Your Submission by contract or applicable
>    law. If You are signing this Agreement on behalf of Your employer, You represent and warrant that You
>    have the necessary authority to bind the listed employer to the obligations contained in this Agreement.
>    You are not expected to provide support for Your Submission, unless You choose to do so. UNLESS
>    REQUIRED BY APPLICABLE LAW OR AGREED TO IN WRITING, AND EXCEPT FOR THE WARRANTIES
>    EXPRESSLY STATED IN SECTIONS 3, 4, AND 6, THE SUBMISSION PROVIDED UNDER THIS AGREEMENT IS
>    PROVIDED WITHOUT WARRANTY OF ANY KIND, INCLUDING, BUT NOT LIMITED TO, ANY WARRANTY OF
>    NONINFRINGEMENT, MERCHANTABILITY, OR FITNESS FOR A PARTICULAR PURPOSE.
> 7. **Notice to Microsoft**. You agree to notify Microsoft in writing of any facts or circumstances of which
>    You later become aware that would make Your representations in this Agreement inaccurate in any
>    respect.
> 8. **Information about Submissions**. You agree that contributions to Projects and information about
>    contributions may be maintained indefinitely and disclosed publicly, including Your name and other
>    information that You submit with Your Submission.
> 9. **Governing Law/Jurisdiction**. This Agreement is governed by the laws of the State of Washington, and
>    the parties consent to exclusive jurisdiction and venue in the federal courts sitting in King County,
>    Washington, unless no federal subject matter jurisdiction exists, in which case the parties consent to
>    exclusive jurisdiction and venue in the Superior Court of King County, Washington. The parties waive all
>    defenses of lack of personal jurisdiction and forum non-conveniens.
> 10. **Entire Agreement/Assignment**. This Agreement is the entire agreement between the parties, and
>     supersedes any and all prior agreements, understandings or communications, written or oral, between
>     the parties relating to the subject matter hereof. This Agreement may be assigned by Microsoft.

**Comment by moneal37** [comment]
@JasonWHowell sorry, it should be signed now. I didn't realize I needed to quote the reply in my signing. I cleaned my other attempts up from the comments so it should just be signed now.

**Comment by cmcclister** [comment]
@bezhan-msft please review this at your eariest convenience so that we can move forward with merging this PR. Thanks!

**Review by bezhan-msft — APPROVED** [review]
Did a quick review - looks fine to me

**Comment by JasonWHowell** [comment]
@DougKlopfenstein This change might impact the file renames you did recently. I'm not sure if Github.com is smart enough to handle the rename seamlessly.

**Comment by DougKlopfenstein** [comment]
@moneal37 Due to some rule changes about how filenames are used in Microsoft docs, there are now conflicts between the naming conventions for your file and folder names. File and folder names now need to be all lowercase, with dashes between words. Could you change the connector filename in your fork to assemble-views.md, which is what it is currently set to in Microsoft:Docs (I'm assuming that would be correct - however, I've seen the connector spelled out as AssembleViews and Assemble Views). That change should solve part of the conflict. In addition, you may need to do a git pull from MicrosoftDocs:main to get the "Connectors" folder changed to "connectors".

If you have any questions, please let me know.

**Comment by moneal37** [comment]
Thanks for reaching out Doug. I'll take a look at this today.

Mike
________________________________

**Comment by moneal37** [comment]
@DougKlopfenstein that proved to be a bit of a mess, but I believe I finally have the files moved to the new names correctly. Note that in addition to renaming, I have also updated the assemble-views.md file to support an updated connector version that was just uploaded.

**Comment by DougKlopfenstein** [comment]
Hi @moneal37 
Everything seems to be ready to publish. As the author of this pull request, you need to leave a comment that says "sign-off" with a pound sign (#) before "sign-off" and no space. That tells the build engine to begin publishing. (If I write out the whole sign off in this comment then things could go awry.)
Thanks

**Comment by moneal37** [comment]
> Hi @moneal37 Everything seems to be ready to publish. As the author of this pull request, you need to leave a comment that says "sign-off" with a pound sign (#) before "sign-off" and no space. That tells the build engine to begin publishing. (If I write out the whole sign off in this comment then things could go awry.) Thanks

Thanks Doug! Do I go ahead and do that even though the connector isn't approved yet? The new doc will need to go live when it does.

**Comment by DougKlopfenstein** [comment]
@moneal37 If the connector is not yet live, you'll need to wait until it is approved and live to do the sign off. The sign off is a signal to us that the connector is live and ready for documentation.

**Comment by ShannonLeavitt** [comment]
@DougKlopfenstein @moneal37 - The content in this PR is complex enough that it needs to be added via the private repo so that it can get automated checks. Let us know if you need help moving the content.

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by moneal37** [comment]
> @DougKlopfenstein @moneal37 - The content in this PR is complex enough that it needs to be added via the private repo so that it can get automated checks. Let us know if you need help moving the content.
> 
> #label:"aq-pr-triaged" @MicrosoftDocs/public-repo-pr-review-team

@ShannonLeavitt I'm not sure what you mean. This is just a minor doc update with some images to support the workflow change. The PR is from my external repo and did go through automated checks. It is just waiting on the connector to get updated. Is there something else we need to do here?

**Comment by JasonWHowell** [comment]
@ShannonLeavitt 
Preview PR: https://github.com/MicrosoftDocs/powerquery-docs-pr/pull/796
Preview page: https://review.learn.microsoft.com/en-us/power-query/connectors/assemble-views?branch=pr-en-us-796

**Comment by JasonWHowell** [comment]
@moneal37 thanks for your contributions. One thing I noticed. There is an image file that isn't used. Is it supposed to be included? assemble-views-append-multiple-1.png

**Comment by JasonWHowell** [comment]
Preview: [Assemble Views connector preview.pdf](https://github.com/MicrosoftDocs/powerquery-docs/files/10963371/Assemble.Views.connector.preview.pdf)

File: powerquery-docs/connectors/assemble-views.md
Comment (Court72) [inline powerquery-docs/connectors/assemble-views.md:80]:
```suggestion
   ![Screenshot that shows where to select properties to include.](media/assemble-views/assemble-views-property-selection.png)
```

File: powerquery-docs/connectors/assemble-views.md
Comment (Court72) [inline powerquery-docs/connectors/assemble-views.md:76]:
```suggestion
   - **Active version only** – Loads only the active version of the model.
   - **All versions** – Loads all versions of the model.
   - **All except active version** – Loads all previous versions of the model without loading the active version (intended for advanced workflows when previous version data only needs to be loaded once and not included in a refresh).
   - **Specific versions** – Loads all specific versions of the model that are selected (specific versions will be selected in the Version Name and Number drop down).
```

File: powerquery-docs/connectors/assemble-views.md
Comment (Court72) [inline powerquery-docs/connectors/assemble-views.md:66]:
```suggestion
   ![Screenshot that shows where to select models to include.](media/assemble-views/assemble-views-model-selection.png)
```

File: powerquery-docs/connectors/assemble-views.md
Comment (Court72) [inline powerquery-docs/connectors/assemble-views.md:60]:
```suggestion
   ![Screenshot that shows where to find your project and expand it.](media/assemble-views/assemble-views-project-selection.png)
```

File: powerquery-docs/connectors/assemble-views.md
Comment (Court72) [inline powerquery-docs/connectors/assemble-views.md:58]:
```suggestion
   - **Views** - fetches all visible columns in a previously saved view.
```

File: powerquery-docs/connectors/assemble-views.md
Comment (Court72) [inline powerquery-docs/connectors/assemble-views.md:56]:
```suggestion
   - **Models** (New!) - fetches select properties from any or all versions of a model.
```

**Comment by Court72** [comment]
@DougKlopfenstein @moneal37 @JasonWHowell 

I've reviewed the staged commits in https://github.com/MicrosoftDocs/powerquery-docs-pr/pull/796 and made some changes per PR review.

Note that **powerquery-docs/connectors/media/assemble-views/assemble-views-append-multiple-1.png** wasn't used in the PR and should be deleted.

This PR is ready for merge (the PR review doesn't have merge perms in this repo).

**Review by ShannonLeavitt — APPROVED** [review]

**Comment by DougKlopfenstein** [comment]
Hi @moneal37 
Sorry for all the confusion. There appears to be an image file that's no longer used in the article (assemble-views-append-multiple-1.png). This image file should be removed from the PR. 
Once that is done, just do a sign off as was discussed earlier in this conversation when the connector update is live.

**Comment by bezhan-msft** [comment]
#sign-off

**Pull request merged**
